docs/src/snap/howto/epa.md

```diff
@@ -0,0 +1,1146 @@
+```
+
+The virtual functions are created on the PCI bus, which can also be verified:
+
+```
+lspci | grep Virtual
+```
+
+...should list the presence of the virtual functions:
+
+```
+98:11.0 Ethernet controller: Intel Corporation Ethernet Adaptive Virtual Function (rev 02)
+98:11.1 Ethernet controller: Intel Corporation Ethernet Adaptive Virtual Function (rev 02)
+98:11.2 Ethernet controller: Intel Corporation Ethernet Adaptive Virtual Function (rev 02)
+...
+99:00.5 Ethernet controller: Intel Corporation Ethernet Adaptive Virtual Function (rev 02)
+99:00.6 Ethernet controller: Intel Corporation Ethernet Adaptive Virtual Function (rev 02)
+99:00.7 Ethernet controller: Intel Corporation Ethernet Adaptive Virtual Function (rev 02)
+```
+
+Examine a specific VF from the list:
+
+```
+lspci -s 98:1f.2 -vv
+```
+
+The output should confirm the correct kernel drivers in use:
+
+```
+98:1f.2 Ethernet controller: Intel Corporation Ethernet Adaptive Virtual Function (rev 02)
+	Subsystem: Intel Corporation Ethernet Adaptive Virtual Function
+	Control: I/O- Mem- BusMaster- SpecCycle- MemWINV- VGASnoop- ParErr- Stepping- SERR- FastB2B- DisINTx-
+	Status: Cap+ 66MHz- UDF- FastB2B- ParErr- DEVSEL=fast >TAbort- <TAbort- <MAbort- >SERR- <PERR- INTx-
+	NUMA node: 1
+	IOMMU group: 391
+	Region 0: Memory at d6e40000 (64-bit, prefetchable) [virtual] [size=128K]
+	Region 3: Memory at d81e8000 (64-bit, prefetchable) [virtual] [size=16K]
+	Capabilities: <access denied>
+	Kernel driver in use: vfio-pci
+	Kernel modules: iavf
+```
+
+Now, create a test pod that will claim a network interface from the DPDK
+network:
+
+```
+cat <<EOF | sudo k8s kubectl apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sriov-test-pod
+  annotations:
+    k8s.v1.cni.cncf.io/networks: dpdk-net1 
+spec:
+  containers:
+    - name: sriov-test-container
+      image: ubuntu:latest
+      resources:
+        limits:
+          cpu: "10"
+          memory: "4000Mi"
+          intel.com/intel_sriov_dpdk: '1'
+          hugepages-1Gi: "16Gi"
+        requests:
+          cpu: "10"
+          memory: "4000Mi"
+          hugepages-1Gi: "16Gi"
+          intel.com/intel_sriov_dpdk: '1'
+      securityContext:
+        capabilities:
+          add:
+            - NET_ADMIN
+            - CAP_SYS_NICE
+            - SYS_NICE
+            - IPC_LOCK
+            - NET_ADMIN
+            - SYS_TIME
+            - CAP_NET_RAW
+            - CAP_BPF
+            - CAP_SYS_ADMIN
+            - SYS_ADMIN
+        privileged: true
+      command: ["bash", "-c", "sleep infinity"]
+EOF
+```
+
+Finally, describe the pod to confirm the DPDK network assignment and also the
+virtual function PCI ID (in this case, `0000:98:1f.2`) that was assigned
+automatically to the `net1` interface:
+
+```
+sudo k8s kubectl describe pod sriov-test-pod
+```
+
+The output (shown truncated below) should indicate the interface and
+the correct PCI address:
+
+```
+...
+                  k8s.v1.cni.cncf.io/network-status:
```

Comment:
I guess those tabs/intendations are here to show that this belongs to a larger output?
I think we can remove them - we already indicate that with the 3 points.